sub-pages/_single-grandchild-pages.html

```diff
@@ -0,0 +1,76 @@
+{% import "_vars-grandchild-pages.html" as vars with context %}
+{% extends "layout-2-1-bleedbar.html" %}
+
+{% set _breadcrumb_items = [('/offices/' + vars.office.slug + '/', vars.office.slug, vars.office.title),
+                           ('/sub-pages/' + vars.parent_sub_page.slug + '/',
+                             vars.parent_sub_page.slug, vars.parent_sub_page.title)]
+%}
+
+{% block pre_content_main -%}
+    {% import "breadcrumbs.html" as breadcrumbs with context %}
+    {{ breadcrumbs.render(_breadcrumb_items) }}
+{% endblock -%}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom
+                  sub-page
+                  sub-page__grandchild
+{%- endblock %}
+
+{% block title -%}
+    {{ sub_page.title | safe if sub_page.title }}
+{%- endblock %}
+
+{% block content_main %}
+    {% if sub_page.title %}
+        <h1>{{ sub_page.title | safe }}</h1>
+    {% endif %}
+
+    {% if sub_page.content %}
+        <section class="block
+                        block__flush-top
+                        block__flush-bottom">
+            {{ sub_page.content | safe }}
+        </section>
+    {% endif %}
+
+    {% if sub_page.body_content %}
+        <section class="block
+                        block__padded-top
+                        block__border-top">
+
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/01dd47a2574106da6a469140a72c30134746c265